tilelang/autodd.py

```
@@ -0,0 +1,1170 @@
+        return cls(source, *args, **kwargs)
+
+    @override
+    def task_generator(self) -> Iterable[Task]:
+        for task in self.generator():
+            new_lines = [line for idx, line in enumerate(self.lines) if idx not in task.applied]
+            source = "\n".join(new_lines)
+            try:
+                ast.parse(source)
+            except Exception as _:
+                # self.update(task, is_interesting=False)
+                continue
```

Ruff (0.14.10)

`try`-`except`-`continue` detected, consider logging the exception

(S112)